File: C#/1009.cs
using System;

class Program
{
    static void Main()
    {
        // Lê o nome
        string nome = Console.ReadLine();
        
        // Lê os valores double
        double n = double.Parse(Console.ReadLine());
        double m = double.Parse(Console.ReadLine());
        
        // Calcula o salário
        double sal = n + (m * 0.15);
        
        // Exibe o resultado formatado
        Console.WriteLine($"TOTAL = R$ {sal:F2}");
    }
}

File: C/1. Beginner/1009.c
#include <stdio.h>

int main() {
    char nome[100];  
    scanf("%99s", nome);  

    double n, m;
    scanf("%lf", &n);  
    scanf("%lf", &m);  
    
    double sal = n + (m * 0.15); 

    printf("TOTAL = R$ %.2lf\n", sal);
    return 0;
}